net-libs/neon/neon-0.32.2.ebuild

```diff
@@ -15,7 +15,8 @@ KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~riscv ~s390 ~
 IUSE="doc expat gnutls kerberos libproxy nls pkcs11 ssl static-libs zlib"
 RESTRICT="test"
```

Comment:
Bug reference/explanation for tests?

Reply:
https://bugs.gentoo.org/691702

So there is 1 failure with `USE="gnutls pkcs11"` + `net-libs/gnutls-3.7.3` (and that test passes with `USE="-gnutls pkcs11"` + `dev-libs/openssl-1.1.1m`, and is skipped with `USE="-pkcs11"`).

Reply:
Can we selectively run tests? Ideally this should also be reported upstream, if it's broken for now just add a bug reference in the ebuild.

Reply:
https://github.com/notroj/neon/issues/72